crates/forge-runner/src/coverage_api.rs

```diff
@@ -0,0 +1,42 @@
+use anyhow::{Context, Result};
+use shared::command::CommandExt;
+use std::process::Stdio;
+use std::{env, fs, path::PathBuf, process::Command};
+
+pub const COVERAGE_DIR: &str = "coverage";
+pub const OUTPUT_FILE_NAME: &str = "coverage.lcov";
```

Comment:
Not sure to be honest. If we add a timestamp, it could result in a bunch of trash files in the coverage directory. Perhaps it would be more effective to include a hash of the executed test names in the output file name?

Reply:
Would the hash of the trace files be better? As this is what determines the behavior of the program, we could cache runs thanks to this.

Reply:
This is a nice improvement, but perhaps should be addresses as a separate issue?

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/2407